packages/@webex/plugin-meetings/src/reconnection-manager/index.ts

```diff
@@ -443,17 +443,19 @@ export default class ReconnectionManager {
       }
     }
 
-    try {
-      LoggerProxy.logger.info(
-        'ReconnectionManager:index#executeReconnection --> Updating meeting data from server.'
-      );
-      await this.webex.meetings.syncMeetings();
-    } catch (syncError) {
-      LoggerProxy.logger.info(
-        'ReconnectionManager:index#executeReconnection --> Unable to sync meetings, reconnecting.',
-        syncError
-      );
-      throw new NeedsRetryError(syncError);
+    if (!this.webex.credentials.isUnverifiedGuest) {
+      try {
+        LoggerProxy.logger.info(
+          'ReconnectionManager:index#executeReconnection --> Updating meeting data from server.'
+        );
+        await this.webex.meetings.syncMeetings();
```

Comment:
question: Or instead of that, could we perhaps enhance syncMeetings such that if it fails it doesn't completely bork the rest of the flow? probably not due to the catch below, but a thought that maybe sync knows when it can and cannot sync meetings instead.

Reply:
Possibly, but hard to do that without also changing the surrounding logging in this function. You don't want the logs to say "Updating meeting data from server." when syncMeetings is actually a no-op.